automation_tools/__init__.py

```diff
@@ -1342,7 +1348,10 @@ def product_install(distribution, create_vm=False, certificate_url=None,
                 execute(setup_oscap, host=host)
             if satellite_version in ('6.2', ''):
                 execute(oscap_content, host=host)
-            execute(enable_ostree, host=host)
+        # ostree plugin is for Sat6.2+ rhel7 only
+        if satellite_version not in ('6.0', '6.1'):
```

Comment:
@lpramuk you'll have to rebase this PR, but as you do that, consider whether the `enable_ostree` task should be executed on `downstream`, `upstream` or both?